content/_posts/falix/dashboard/2024-07-26-subusers.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: Learn how to manage your Subusers
+category: Dashboard
+tags: Server
+permalink: falix/dashboard/server/subusers
+description: Learn how to manage and create Subusers.
+author: TheTWIXhunter & Duolingo

```

Comment:
> & Duolingo

This will cause confusion

Reply:
You shoud ask to duogone how he wants to resolve this

Reply:
I woud think there is no copyright and confusion if you refer to @tp-duolingo as duogone but I am not sure if that works with ducky’s system

Reply:
fixed by ducky's nickname system hopefully